public/index.html

```diff
@@ -13,6 +13,9 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/app-icon.png" />
     <script src="/config.js"></script>
+    <% if (process.env.REACT_APP_USE_LOCAL_CONFIG) { %>
```

Comment:
I initially used `NODE_ENV === "development"` but then if you don't need a config.local.js this will result in an error when it tries to load a file that doesn't exist.

Reply:
Yes very good point! I'm doing the same thing stash the file `config.js` often.
The downside of having a new env variable is that we have to add it locally, I found a way to make this work automatically that may work as well:
```html
    <% if (process.env.NODE_ENV === "development" &&
    require("fs").existsSync("public/config.local.js")) { %>
...
```

Reply:
Thanks! That's much nice, I'll update it.